pkg/reconciler/kubernetes/tektoninstallerset/install.go

```diff
@@ -128,17 +128,22 @@ func isClusterScoped(kind string) bool {
 
 func (i *installer) ensureResources(resources []unstructured.Unstructured) error {
 	for _, r := range resources {
+		ressourceLogger := i.logger.With(
+			"kind", r.GetKind(),
+			"namespace", r.GetNamespace(),
+			"name", r.GetName(),
+		)
 		expectedHash, err := hash.Compute(r.Object)
 		if err != nil {
+			ressourceLogger.Error("failed to compute resource hash", "error", err)
 			return err
 		}
-
-		i.logger.Infof("fetching resource %s: %s/%s", r.GetKind(), r.GetNamespace(), r.GetName())
+		ressourceLogger.Debug("fetching resource")
 
 		res, err := i.mfClient.Get(&r)
 		if err != nil {
 			if apierrs.IsNotFound(err) {
-				i.logger.Infof("resource not found, creating %s: %s/%s", r.GetKind(), r.GetNamespace(), r.GetName())
+				ressourceLogger.Info("creating new resource")
```

Comment:
Just had a suggestion, any possibility we can specify which resource we are creating, I think it would be much more descriptive
wdyt ?

Reply:
https://github.com/tektoncd/operator/pull/2674/files#diff-d33a60db8d53ff9977610e53a41e1b78c22dee3ac5b0164487273b7979af38dbR131-R135